CMakeLists.txt

```diff
@@ -4,6 +4,8 @@
 # Ubuntu LTS release (Xenial). Fedora, CentOS (with cmake3), Homebrew,
 # and Chocolatey all provided at least 3.5.1 as well.
 cmake_minimum_required(VERSION 3.5.1)
+set(CMAKE_BUILD_TYPE Release)
```

Comment:
Put these immediately after the `project` definition, and add a comment explaining why they're necessary.